regtests/ww3_tp2.15/input_flx5/ww3_prnc_RHO.inp

```diff
@@ -0,0 +1,50 @@
+$ -------------------------------------------------------------------- $
+$ WAVEWATCH III Field preprocessor input file                          $
+$ -------------------------------------------------------------------- $
+$ Mayor types of field and time flag
+$   Field types  :  ICE   Ice concentrations.
+$                   LEV   Water levels.
+$                   WND   Winds.
+$                   WNS   Winds (including air-sea temp. dif.)
+$                   CUR   Currents.
+$                   DAT   Data for assimilation.
+$
```

Comment:
Here, please define RHO

Reply:
defined in ww3_prnc_RHO.* and ww3_prnc_TAU.*